src/xcode/ENA/ENA/Source/Extensions/ExposureSubmissionParsable.swift

```diff
@@ -0,0 +1,99 @@
+//
+// Corona-Warn-App
+//
+// SAP SE and all other contributors
+// copyright owners license this file to you under the Apache
+// License, Version 2.0 (the "License"); you may not use this
+// file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+import Foundation
+import ExposureNotification
+
+// MARK: - ExposureSubmissionErrorTransformable protocol.
+
+/// This protocol ensures that a given ErrorType can be transformed into an
+/// `ExposureSubmissionError`.
+/// For the future, if other transformations are needed, it is advised to create
+/// a corrseponding protocol specific to the destination error type.
+protocol ExposureSubmissionErrorTransformable {
+	func transformToExposureSubmissionError() -> ExposureSubmissionError
```

Comment:
maybe drop the "transform" prefix and just go with `toExposureSubmissionError`? Makes the API a bit shorter without losing any information

Reply:
will do.